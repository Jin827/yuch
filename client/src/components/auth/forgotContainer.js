import React from 'react';
import { Formik } from 'formik';
/* --- Components --- */
import { admin } from '../../data/data';
import ForgotForm from './forgotForm';
import { forgotValidation } from './formValidation';

const ForgotContainer = () => {
  const handleConfirmEmail = () =>
    console.log('handleConfirmEmail button clicked');

  const values = {
    contactNo: '',
    email: '',
  };
  return (
    <div className="forgot-container">
      <p className="mb2 b f-regular">
        {admin.companyName}에 등록되어 있는 이메일 주소를 입력해 주세요.
      </p>
      <p className="c-text-grey lh-2 f-mini">
        고객님이 입력한 이메일로 인증코드가 전송될 것 입니다. <br />
        등록된 이메일 주소가 없거나, 기억이 나지 않으시면 {admin.companyName}
        으로 문의 바랍니다.
      </p>
      <Formik
        initialValues={values}
        render={props => <ForgotForm {...props} />}
        onSubmit={handleConfirmEmail}
        validationSchema={forgotValidation}
      />
      <ForgotForm />
    </div>
  );
};

export default ForgotContainer;
