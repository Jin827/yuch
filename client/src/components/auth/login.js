import React from 'react';

const Login = () => (
  <div className="tc login-container">
    <h1>Comming Soon...</h1>
  </div>
);
export default Login;
