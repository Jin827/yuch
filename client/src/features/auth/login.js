import React from 'react';
import TextFields from './textFields';
import * as data from '../../shared/data';
import Buttons from './buttons';

const Login = () => (
  <div className="tc login-container">
    <TextFields loginform={data.loginForm} />
    <Buttons loginbtn={data.loginBtn} />
  </div>
);
export default Login;
