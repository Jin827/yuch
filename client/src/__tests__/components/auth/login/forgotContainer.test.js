import { ForgotContainer } from '../../../../components/auth/login/forgotContainer';
import React, { render, cleanup } from '../../../setupTests';

afterEach(cleanup);

const defaultProps = {
  authActions: {
    findUsernameWithEmail: jest.fn(),
    findUsernameWithContact: jest.fn(),
    sendVerificationCodeToEmail: jest.fn(),
  },
  messageActions: { addFlashMessage: jest.fn() },
};

const setUp = (props = {}) => {
  const setupProps = { ...defaultProps, ...props };
  const component = render(<ForgotContainer {...setupProps} />);
  return { component };
};

describe('<forgotContainer />', () => {
  it('renders UsernameFormBox Component when user clicks forgot username button', () => {
    const { component } = setUp({ location: { search: '?value=username' } });
    expect(component).toMatchSnapshot();
  });

  it('renders PasswordFormBox Component when user clicks forgot password button', () => {
    const { component } = setUp({ location: { search: '?value=password' } });
    expect(component).toMatchSnapshot();
  });
});
