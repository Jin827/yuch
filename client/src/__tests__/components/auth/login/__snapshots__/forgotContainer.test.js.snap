// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<forgotContainer /> renders PasswordFormBox Component when user clicks forgot password button 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="verify-user--container"
      >
        <div
          class="tc"
        >
          <p
            class="mb2 b f-regular lh-2"
          >
            고객님의 아이디와 
            유청
            에 등록되어 있는 이메일 주소를 입력해 주세요.
          </p>
          <p
            class="c-text-grey f-mini"
          >
            고객님이 입력한 이메일로 인증코드가 전송됩니다.
          </p>
          <p
            class="c-text-grey f-mini"
          >
            비밀번호를 찾는데 어려움이 있으면 
            유청
            으로 문의 바랍니다.
          </p>
          <form
            class="flex flex-column-m items-center mt4"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation1 verify-user--paper MuiPaper-rounded"
            >
              <table>
                <tbody>
                  <tr>
                    <td
                      class="verify-user--label"
                    >
                      아이디
                    </td>
                    <td>
                      <div
                        class="MuiFormControl-root MuiTextField-root Input-textFieldA-209 MuiFormControl-marginNormal"
                      >
                        
                        <div
                          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
                        >
                          <div
                            class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                          >
                            <svg
                              class="svg-icon icon-filledUser || ''"
                              height="30"
                              viewBox="0 0 30 30"
                              width="30"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M9 11.75c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25 1.25-.56 1.25-1.25-.56-1.25-1.25-1.25zm6 0c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25 1.25-.56 1.25-1.25-.56-1.25-1.25-1.25zM12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8 0-.29.02-.58.05-.86 2.36-1.05 4.23-2.98 5.21-5.37C11.07 8.33 14.05 10 17.42 10c.78 0 1.53-.09 2.25-.26.21.71.33 1.47.33 2.26 0 4.41-3.59 8-8 8z"
                              />
                              <path
                                d="M0 0h24v24H0z"
                                fill="none"
                              />
                            </svg>
                          </div>
                          <input
                            aria-invalid="false"
                            class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                            data-testid="username"
                            name="username"
                            placeholder="yucheong"
                            required=""
                            tabindex="username"
                            type="text"
                            value=""
                          />
                          <fieldset
                            aria-hidden="true"
                            class="PrivateNotchedOutline-root-260 MuiOutlinedInput-notchedOutline"
                            style="padding-left: 8px;"
                          >
                            <legend
                              class="PrivateNotchedOutline-legend-261"
                              style="width: 0px;"
                            >
                              <span>
                                ​
                              </span>
                            </legend>
                          </fieldset>
                        </div>
                      </div>
                    </td>
                  </tr>
                  <tr>
                    <td
                      class="verify-user--hr"
                    />
                  </tr>
                  <tr>
                    <td
                      class="verify-user--label"
                    >
                      이메일
                    </td>
                    <td>
                      <div
                        class="MuiFormControl-root MuiTextField-root Input-textFieldA-209 MuiFormControl-marginNormal"
                      >
                        
                        <div
                          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
                        >
                          <div
                            class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                          >
                            <svg
                              class="svg-icon icon-email || ''"
                              height="30"
                              viewBox="0 0 30 30"
                              width="30"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M0 0h24v24H0V0z"
                                fill="none"
                              />
                              <path
                                d="M22 6c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6zm-2 0l-8 5-8-5h16zm0 12H4V8l8 5 8-5v10z"
                              />
                            </svg>
                          </div>
                          <input
                            aria-invalid="false"
                            class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                            data-testid="email"
                            name="email"
                            placeholder="sleket12@hanmail.net"
                            required=""
                            tabindex="email"
                            type="email"
                            value=""
                          />
                          <fieldset
                            aria-hidden="true"
                            class="PrivateNotchedOutline-root-260 MuiOutlinedInput-notchedOutline"
                            style="padding-left: 8px;"
                          >
                            <legend
                              class="PrivateNotchedOutline-legend-261"
                              style="width: 0px;"
                            >
                              <span>
                                ​
                              </span>
                            </legend>
                          </fieldset>
                        </div>
                      </div>
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
            <button
              class="MuiButtonBase-root MuiButton-root login-btn FormButton-button-262 MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall"
              tabindex="0"
              type="submit"
            >
              <span
                class="MuiButton-label"
              >
                계속하기
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </form>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="verify-user--container"
    >
      <div
        class="tc"
      >
        <p
          class="mb2 b f-regular lh-2"
        >
          고객님의 아이디와 
          유청
          에 등록되어 있는 이메일 주소를 입력해 주세요.
        </p>
        <p
          class="c-text-grey f-mini"
        >
          고객님이 입력한 이메일로 인증코드가 전송됩니다.
        </p>
        <p
          class="c-text-grey f-mini"
        >
          비밀번호를 찾는데 어려움이 있으면 
          유청
          으로 문의 바랍니다.
        </p>
        <form
          class="flex flex-column-m items-center mt4"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation1 verify-user--paper MuiPaper-rounded"
          >
            <table>
              <tbody>
                <tr>
                  <td
                    class="verify-user--label"
                  >
                    아이디
                  </td>
                  <td>
                    <div
                      class="MuiFormControl-root MuiTextField-root Input-textFieldA-209 MuiFormControl-marginNormal"
                    >
                      
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
                      >
                        <div
                          class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                        >
                          <svg
                            class="svg-icon icon-filledUser || ''"
                            height="30"
                            viewBox="0 0 30 30"
                            width="30"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M9 11.75c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25 1.25-.56 1.25-1.25-.56-1.25-1.25-1.25zm6 0c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25 1.25-.56 1.25-1.25-.56-1.25-1.25-1.25zM12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8 0-.29.02-.58.05-.86 2.36-1.05 4.23-2.98 5.21-5.37C11.07 8.33 14.05 10 17.42 10c.78 0 1.53-.09 2.25-.26.21.71.33 1.47.33 2.26 0 4.41-3.59 8-8 8z"
                            />
                            <path
                              d="M0 0h24v24H0z"
                              fill="none"
                            />
                          </svg>
                        </div>
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                          data-testid="username"
                          name="username"
                          placeholder="yucheong"
                          required=""
                          tabindex="username"
                          type="text"
                          value=""
                        />
                        <fieldset
                          aria-hidden="true"
                          class="PrivateNotchedOutline-root-260 MuiOutlinedInput-notchedOutline"
                          style="padding-left: 8px;"
                        >
                          <legend
                            class="PrivateNotchedOutline-legend-261"
                            style="width: 0px;"
                          >
                            <span>
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </td>
                </tr>
                <tr>
                  <td
                    class="verify-user--hr"
                  />
                </tr>
                <tr>
                  <td
                    class="verify-user--label"
                  >
                    이메일
                  </td>
                  <td>
                    <div
                      class="MuiFormControl-root MuiTextField-root Input-textFieldA-209 MuiFormControl-marginNormal"
                    >
                      
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
                      >
                        <div
                          class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                        >
                          <svg
                            class="svg-icon icon-email || ''"
                            height="30"
                            viewBox="0 0 30 30"
                            width="30"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M0 0h24v24H0V0z"
                              fill="none"
                            />
                            <path
                              d="M22 6c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6zm-2 0l-8 5-8-5h16zm0 12H4V8l8 5 8-5v10z"
                            />
                          </svg>
                        </div>
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                          data-testid="email"
                          name="email"
                          placeholder="sleket12@hanmail.net"
                          required=""
                          tabindex="email"
                          type="email"
                          value=""
                        />
                        <fieldset
                          aria-hidden="true"
                          class="PrivateNotchedOutline-root-260 MuiOutlinedInput-notchedOutline"
                          style="padding-left: 8px;"
                        >
                          <legend
                            class="PrivateNotchedOutline-legend-261"
                            style="width: 0px;"
                          >
                            <span>
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
          <button
            class="MuiButtonBase-root MuiButton-root login-btn FormButton-button-262 MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall"
            tabindex="0"
            type="submit"
          >
            <span
              class="MuiButton-label"
            >
              계속하기
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </form>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`<forgotContainer /> renders UsernameFormBox Component when user clicks forgot username button 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="verify-user--container"
      >
        <div
          class="tc"
        >
          <p
            class="mb2 b f-regular lh-2"
          >
            유청
            에 등록되어 있는 이메일주소 혹은 연락처를 입력해 주세요.
          </p>
          <p
            class="c-text-grey f-mini mb3"
          >
            이메일주소 혹은 연락처를 등록하지 않았거나, 기억이 나지 않으면 
            유청
            으로 문의 바랍니다.
          </p>
          <form
            class="flex flex-column-m items-center mt4"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation1 verify-user--paper MuiPaper-rounded"
            >
              <table>
                <tbody>
                  <tr
                    class="verify-user--bb"
                  >
                    <td
                      class="verify-user--label"
                    >
                      인증방법
                    </td>
                    <td
                      class="pw1"
                    >
                      <div
                        aria-label="VerifyUserOptions"
                        class="MuiFormGroup-root ph MuiFormGroup-row"
                        role="radiogroup"
                      >
                        <label
                          class="MuiFormControlLabel-root"
                        >
                          <span
                            aria-disabled="false"
                            class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-41 MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-checked-42 Mui-checked MuiIconButton-colorPrimary"
                          >
                            <span
                              class="MuiIconButton-label"
                            >
                              <div
                                class="PrivateRadioButtonIcon-root-57 PrivateRadioButtonIcon-checked-59"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root"
                                  focusable="false"
                                  role="presentation"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                                  />
                                </svg>
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-58"
                                  focusable="false"
                                  role="presentation"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                                  />
                                </svg>
                              </div>
                              <input
                                checked=""
                                class="PrivateSwitchBase-input-44"
                                name="contactNo"
                                type="radio"
                                value="contactNo"
                              />
                            </span>
                            <span
                              class="MuiTouchRipple-root"
                            />
                          </span>
                          <span
                            class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                          >
                            휴대폰번호
                          </span>
                        </label>
                        <label
                          class="MuiFormControlLabel-root"
                          data-testid="email-forgot"
                        >
                          <span
                            aria-disabled="false"
                            class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-41 MuiRadio-root MuiRadio-colorPrimary MuiIconButton-colorPrimary"
                          >
                            <span
                              class="MuiIconButton-label"
                            >
                              <div
                                class="PrivateRadioButtonIcon-root-57"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root"
                                  focusable="false"
                                  role="presentation"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                                  />
                                </svg>
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-58"
                                  focusable="false"
                                  role="presentation"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                                  />
                                </svg>
                              </div>
                              <input
                                class="PrivateSwitchBase-input-44"
                                name="contactNo"
                                type="radio"
                                value="email"
                              />
                            </span>
                            <span
                              class="MuiTouchRipple-root"
                            />
                          </span>
                          <span
                            class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                          >
                            이메일
                          </span>
                        </label>
                      </div>
                    </td>
                  </tr>
                  <tr>
                    <td
                      class="verify-user--hr"
                    />
                  </tr>
                  <tr>
                    <td
                      class="verify-user--label"
                    >
                      연락처
                    </td>
                    <td>
                      <div
                        class="MuiFormControl-root MuiTextField-root Input-textFieldA-100 MuiFormControl-marginNormal"
                      >
                        
                        <div
                          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
                        >
                          <div
                            class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                          >
                            <svg
                              class="svg-icon icon-phone || ''"
                              height="30"
                              viewBox="0 0 30 30"
                              width="30"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M0 0h24v24H0V0z"
                                fill="none"
                              />
                              <path
                                d="M6.54 5c.06.89.21 1.76.45 2.59l-1.2 1.2c-.41-1.2-.67-2.47-.76-3.79h1.51m9.86 12.02c.85.24 1.72.39 2.6.45v1.49c-1.32-.09-2.59-.35-3.8-.75l1.2-1.19M7.5 3H4c-.55 0-1 .45-1 1 0 9.39 7.61 17 17 17 .55 0 1-.45 1-1v-3.49c0-.55-.45-1-1-1-1.24 0-2.45-.2-3.57-.57-.1-.04-.21-.05-.31-.05-.26 0-.51.1-.71.29l-2.2 2.2c-2.83-1.45-5.15-3.76-6.59-6.59l2.2-2.2c.28-.28.36-.67.25-1.02C8.7 6.45 8.5 5.25 8.5 4c0-.55-.45-1-1-1z"
                              />
                            </svg>
                          </div>
                          <input
                            aria-invalid="false"
                            class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                            data-testid="contactNo"
                            name="contactNo"
                            placeholder="054-745-0999"
                            tabindex="contactNo"
                            type="text"
                            value=""
                          />
                          <fieldset
                            aria-hidden="true"
                            class="PrivateNotchedOutline-root-151 MuiOutlinedInput-notchedOutline"
                            style="padding-left: 8px;"
                          >
                            <legend
                              class="PrivateNotchedOutline-legend-152"
                              style="width: 0px;"
                            >
                              <span>
                                ​
                              </span>
                            </legend>
                          </fieldset>
                        </div>
                      </div>
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
            <button
              class="MuiButtonBase-root MuiButton-root login-btn FormButton-button-153 MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall"
              tabindex="0"
              type="submit"
            >
              <span
                class="MuiButton-label"
              >
                계속하기
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </form>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="verify-user--container"
    >
      <div
        class="tc"
      >
        <p
          class="mb2 b f-regular lh-2"
        >
          유청
          에 등록되어 있는 이메일주소 혹은 연락처를 입력해 주세요.
        </p>
        <p
          class="c-text-grey f-mini mb3"
        >
          이메일주소 혹은 연락처를 등록하지 않았거나, 기억이 나지 않으면 
          유청
          으로 문의 바랍니다.
        </p>
        <form
          class="flex flex-column-m items-center mt4"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation1 verify-user--paper MuiPaper-rounded"
          >
            <table>
              <tbody>
                <tr
                  class="verify-user--bb"
                >
                  <td
                    class="verify-user--label"
                  >
                    인증방법
                  </td>
                  <td
                    class="pw1"
                  >
                    <div
                      aria-label="VerifyUserOptions"
                      class="MuiFormGroup-root ph MuiFormGroup-row"
                      role="radiogroup"
                    >
                      <label
                        class="MuiFormControlLabel-root"
                      >
                        <span
                          aria-disabled="false"
                          class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-41 MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-checked-42 Mui-checked MuiIconButton-colorPrimary"
                        >
                          <span
                            class="MuiIconButton-label"
                          >
                            <div
                              class="PrivateRadioButtonIcon-root-57 PrivateRadioButtonIcon-checked-59"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root"
                                focusable="false"
                                role="presentation"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                                />
                              </svg>
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-58"
                                focusable="false"
                                role="presentation"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                                />
                              </svg>
                            </div>
                            <input
                              checked=""
                              class="PrivateSwitchBase-input-44"
                              name="contactNo"
                              type="radio"
                              value="contactNo"
                            />
                          </span>
                          <span
                            class="MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                        >
                          휴대폰번호
                        </span>
                      </label>
                      <label
                        class="MuiFormControlLabel-root"
                        data-testid="email-forgot"
                      >
                        <span
                          aria-disabled="false"
                          class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-41 MuiRadio-root MuiRadio-colorPrimary MuiIconButton-colorPrimary"
                        >
                          <span
                            class="MuiIconButton-label"
                          >
                            <div
                              class="PrivateRadioButtonIcon-root-57"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root"
                                focusable="false"
                                role="presentation"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                                />
                              </svg>
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-58"
                                focusable="false"
                                role="presentation"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                                />
                              </svg>
                            </div>
                            <input
                              class="PrivateSwitchBase-input-44"
                              name="contactNo"
                              type="radio"
                              value="email"
                            />
                          </span>
                          <span
                            class="MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                        >
                          이메일
                        </span>
                      </label>
                    </div>
                  </td>
                </tr>
                <tr>
                  <td
                    class="verify-user--hr"
                  />
                </tr>
                <tr>
                  <td
                    class="verify-user--label"
                  >
                    연락처
                  </td>
                  <td>
                    <div
                      class="MuiFormControl-root MuiTextField-root Input-textFieldA-100 MuiFormControl-marginNormal"
                    >
                      
                      <div
                        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
                      >
                        <div
                          class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                        >
                          <svg
                            class="svg-icon icon-phone || ''"
                            height="30"
                            viewBox="0 0 30 30"
                            width="30"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M0 0h24v24H0V0z"
                              fill="none"
                            />
                            <path
                              d="M6.54 5c.06.89.21 1.76.45 2.59l-1.2 1.2c-.41-1.2-.67-2.47-.76-3.79h1.51m9.86 12.02c.85.24 1.72.39 2.6.45v1.49c-1.32-.09-2.59-.35-3.8-.75l1.2-1.19M7.5 3H4c-.55 0-1 .45-1 1 0 9.39 7.61 17 17 17 .55 0 1-.45 1-1v-3.49c0-.55-.45-1-1-1-1.24 0-2.45-.2-3.57-.57-.1-.04-.21-.05-.31-.05-.26 0-.51.1-.71.29l-2.2 2.2c-2.83-1.45-5.15-3.76-6.59-6.59l2.2-2.2c.28-.28.36-.67.25-1.02C8.7 6.45 8.5 5.25 8.5 4c0-.55-.45-1-1-1z"
                            />
                          </svg>
                        </div>
                        <input
                          aria-invalid="false"
                          class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                          data-testid="contactNo"
                          name="contactNo"
                          placeholder="054-745-0999"
                          tabindex="contactNo"
                          type="text"
                          value=""
                        />
                        <fieldset
                          aria-hidden="true"
                          class="PrivateNotchedOutline-root-151 MuiOutlinedInput-notchedOutline"
                          style="padding-left: 8px;"
                        >
                          <legend
                            class="PrivateNotchedOutline-legend-152"
                            style="width: 0px;"
                          >
                            <span>
                              ​
                            </span>
                          </legend>
                        </fieldset>
                      </div>
                    </div>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
          <button
            class="MuiButtonBase-root MuiButton-root login-btn FormButton-button-153 MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall"
            tabindex="0"
            type="submit"
          >
            <span
              class="MuiButton-label"
            >
              계속하기
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </form>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
